package.json

```diff
@@ -10,6 +10,7 @@
     "prop-types": "^15.8.1",
```

Comment:
- [ ] [OPTIONAL] GOOD JOB 💥👏. I would suggest adding screenshot of website to readme file since design is now COMPLETE 👋!!